typings/index.d.ts

```diff
@@ -1666,12 +1667,13 @@ declare module 'discord.js' {
 		private connection: WebSocket | null;
 		private helloTimeout: NodeJS.Timeout | null;
```

Comment:
This line needs to be edited due to the change in https://github.com/discordjs/discord.js/blob/e47fc1e0f85ee63409979084bd45e0b748899fad/src/client/websocket/WebSocketShard.js#L128
```suggestion
		private helloTimeout: NodeJS.Timeout | undefined;
```